src/tuned_models.jl

```diff
@@ -309,7 +309,12 @@ function TunedModel(args...; model=nothing,
         else
             throw(ERR_MODEL_TYPE)
         end
+    elseif model isa Type && model <: Model
+        msg = "Uninstantiated $model passed; pass an instantiated model so that this" *
+            "package can mutate the hyperparameters specified by the range."
+        throw(AssertionError(msg))
```

Comment:
Since we aim to replace the `Model` type heirarchy in the future, let's drop the test against `Model`.

```suggestion
    elseif model isa Type 
        msg = "Type encountered where model instance expected. (Tuning evaluates models "*
            "generated by mutating clones of provided instance, as specified by `range`.) "
        throw(AssertionError(msg))
```

Even better, for better maintenance, please define the exception as a global constant, and refer to the constant in your `@test_throws` test.

Reply:
Aaaaah! Now I get why those things are constants

I‘ll update the PR tomorrow (23:48 now here)